sda-commons-server-testing/src/main/java/org/sdase/commons/server/testing/SystemPropertyClassExtension.java

```diff
@@ -0,0 +1,74 @@
+package org.sdase.commons.server.testing;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Supplier;
+import org.junit.jupiter.api.extension.AfterAllCallback;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+public class SystemPropertyClassExtension implements BeforeAllCallback, AfterAllCallback {
```

Comment:
I think it would be nice to also have a description and copy/paste template here.